### ISSUE TITLE
Building uvcdat creates build directories in the source directory

### ISSUE THREAD

**Issue by danlipsa** [UV-CDAT/uvcdat#2048]
This creates problems when updating the SHA in your source directory. It is not sufficient to delete the build directory, reconfigure and rebuild. You have to do a 'find . -name build -exec rm -rf {} \;' in the source directory to start fresh.

@doutriaux1 @aashish24 This is not good practice - it would be nice to fix this.

**Comment by aashish24** [comment]
so even with conda this is the case?

**Comment by doutriaux1** [comment]
yes and now, conda always creates a new clone. It's not terribly difficult to change this, I have no objection.
